src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-rate-limit/nfs-rate-limit.component.html

```diff
@@ -336,3 +308,158 @@
     </ng-container>
   </form>
 </ng-container>
+
+<ng-container *ngIf="allowIops">
+  <form name="rateLimitForm"
+        #formDir="ngForm"
+        [formGroup]="rateLimitForm"
+        novalidate>
```

Comment:
why would we need a different form for this? I would say no, and append it to the already existing form element. You can just hide it behind a toggle. Ideally we shouldn't need separate forms for two different things in the same page